### PR TITLE
Changed Gitlab CI runner tag for Windows to windows-inria

### PR DESCRIPTION
**Pull request by MSoegtropIMC**
Since Gitlab started to have their own free runners with tag "windows" and since these are not compatible with our scripts, we had to change the tag for the INRIA runners to "windows-inria". This change adjusts the `.gitlab-ci.yml` file accordingly.

### CODE REVIEW

**Comment by MSoegtropIMC** [comment]
@ppedrot : you need to backport this to the 8.11 branch if you want to run Windows CI on it.

**Review by ejgallego — APPROVED** [review]
Will merge when CI passes. Check on GitLab settings looks OK.

**Comment by MSoegtropIMC** [comment]
> Will merge when CI passes. Check on GitLab settings looks OK.

Since people are waiting for it I think you could even merge now. The windows builkd did pick up the correct runner and this is all what CI can show in this case.

**Comment by MSoegtropIMC** [comment]
@ejgallego : the failure of the 32 bit windows build in the merge CI is something else - possibly my bad. It might be the switch to OCaml 4.08.1. I don't think I properly tested this on 32 bit.

**Pull request merged**
